config/k8s/wait.py

```diff
@@ -32,6 +32,28 @@ def context_valid(c):
     return valid
 
 
+def find_pod(core, args):
+    retries = 10
+    while retries:
+        try:
+            pod = core.list_namespaced_pod(namespace=args.namespace,
+                                           watch=False,
+                                           label_selector="job-name={}".
+                                           format(args.job_name))
+            return(pod)
```

Comment:
return is a keyword not a function 
```suggestion
            return pod
```

Reply:
done